integration_test.go

```diff
@@ -133,124 +33,6 @@ func saveInternalState(t *testing.T, internalState []byte, dataDir string) strin
 	return filename
 }
 
-// testHelperCartographer is responsible for running a viam-cartographer process using the desired mock sensors. Once started it will
-// wait for all data to be processed by monitor sensor channels. After data has been fully processed, the endpoints Position,
-// PointCloudMap, InternalState are evaluated and the process is closed out. The final internal state of cartographer is then returned.
-func testHelperCartographer(
```

Comment:
This was moved to `testhelper/integrationtesthelper.go` and renamed to `IntegrationCartographer`.